hive3/src/main/java/org/apache/iceberg/mr/hive/vector/HiveVectorizedReader.java

```diff
@@ -98,11 +100,13 @@ private HiveVectorizedReader() {
     try {
       switch (format) {
         case ORC:
-          InputSplit split = new OrcSplit(path, null, task.start(), task.length(), (String[]) null, null,
-              false, false, Lists.newArrayList(), 0, task.length(), path.getParent());
-          RecordReader<NullWritable, VectorizedRowBatch> recordReader = null;
 
-          recordReader = new VectorizedOrcInputFormat().getRecordReader(split, job, reporter);
+          OrcTail orcTail = VectorizedReadUtils.getOrcTail(inputFile, job);
```

Comment:
Maybe it'd be useful to have a short comment on why the tail read is necessary here?